models/books.js

```diff
@@ -0,0 +1,13 @@
+var mongoose = require("mongoose");
```

Comment:
Rename this to be `book.js` so it's singular instead of plural. It's only one book blueprint (and we like to stick to naming conventions.